components/layout/menu.hbs

```diff
@@ -1,316 +1,293 @@
-            >Communities</a></li>
-          <li
-            class="submenu-item is-submenu-item is-dropdown-submenu-item"
-            role="none"
-          ><a
-              href="/client-success/"
-              title="Client Successes"
-              role="menuitem"
-            >Client Successes</a></li>
-          <li
-            class="submenu-item is-submenu-item is-dropdown-submenu-item"
-            role="none"
-          ><a
-              href="/our-expertise/partnerships"
-              title="Partnerships"
-              role="menuitem"
-            >Partnerships</a></li>
-        </ul>
-      </li>
-      <li class="top-level-item" role="none"><a
-          title="Contact"
-          href="/contact-us/"
-          role="menuitem"
-        >Contact</a></li>
-    </ul>
-    <div id="country-selector" class="country-selector show-for-large">
-      <ul
-        class="dropdown menu simple"
-        data-dropdown-menu="guqpuq-dropdown-menu"
-        data-closing-time="500"
-        data-hover-delay="200"
-        data-alignment="right"
-        role="menubar"
-      >
-        <li class="top-level-item" role="none"><a
-            title="Login"
-            href="https://my.aquent.com/mat/myaquent?PROC=AWUIDrawMatLogin&amp;locale=en_US"
-            role="menuitem"
-          >Login</a></li>
-        <li
-          class="is-dropdown-submenu-parent opens-left"
-          role="none"
-          aria-haspopup="true"
-          aria-label="USA/Canada"
-        >
-          <a
-            class="down-arrow after"
-            title="USA/Canada"
-            tabindex="0"
-            role="menuitem"
-          >USA/Canada</a>
-          <ul
-            class="submenu menu is-dropdown-submenu first-sub vertical"
-            data-submenu=""
-            role="menubar"
-          >
-            <li role="none" class="is-submenu-item is-dropdown-submenu-item"><a
-                href="https://aquent.com.au"
-                title="Australia"
+
+  <div class="grid-container top-menu">
+    <div class="site-title">
+      <a class="logo" href="/" title="Aquent"><img
+          src="/img/Vitamin-T-Logo.svg"
+        /></a>
+    </div>
+
+    <div class="grid-container navbar" id="navbar">
+      <details>
+        <summary class="tertiary-info--large">
+          <span class="menuitem">
+            <i class="arrow--before"></i>Talents<i class="arrow--after"></i>
+          </span>
+        </summary>
+        <div class="dropdown-menu section--bg-white" role="menubar">
+          <ul class="dropdown-menu-main" role="menubar">
+            <li class="heading" role="none"><span
+                class="p--body-emphasis"
+              >Employment</span></li>
+            <li class="submenu-item" role="none"><a
+                href="#"
+                title="Why work with us"
+                role="menuitem"
+                class="anchor--primary"
+              ><span class="p--body-small">Why work with us</span></a></li>
```

Comment:
I'm a little concerned reading this markup with our BEM methodology. If we presume `top-menu` (line 7) is our block class for the component, I'd like to see us naming our elements more inline at this level, so `top-menu__li--heading`, `top-menu__li--menuitem`, `top-menu__link`, etc.